htdocs/takepos/index.php

```diff
@@ -527,7 +528,8 @@ function ChangeThirdparty(idcustomer) {
 
 function deleteline() {
 	console.log("Delete line");
-	$("#poslines").load("invoice.php?action=deleteline&token=<?php echo newToken(); ?>&place="+place+"&idline="+selectedline, function() {
+	invoiceid = $("#invoiceid").val();
+	$("#poslines").load("invoice.php?action=deleteline&token=<?php echo newToken(); ?>&place="+place+"&idline="+selectedline+"invoiceid="+invoiceid, function() {
```

Comment:
`...line="+selectedline+"invoiceid="+inv...`
looks stange. May be 
`...line="+selectedline+"&invoiceid="+inv...`